habitat_baselines/utils/gym_adapter.py

```diff
@@ -89,10 +89,15 @@ def __init__(self, env, save_orig_obs: bool = False):
         self._gym_achieved_goal_keys = env._rl_config.get(
             "GYM_ACHIEVED_GOAL_KEYS", []
         )
-        self._fix_info_dict = env._rl_config.get("GYM_FIX_INFO_DICT", False)
+        self._fix_info_dict = env._rl_config.get("GYM_FIX_INFO_DICT", True)
         self._gym_action_keys = env._rl_config.get("GYM_ACTION_KEYS", None)
         self._gym_obs_keys = env._rl_config.get("GYM_OBS_KEYS", None)
 
+        if self._gym_obs_keys is None:
```

Comment:
It looks like the source of truth is the config and if there is no `GYM_OBS_KEYS` in the config, then the fallback is the observation space of the environment. Is that the expected behavior? 
If the observation_space is always available, why is there a `GYM_OBS_KEYS` in the config? (It looks redundant).
I think we should always use `env.observation_space.spaces` and throw an error if it is different from the config provided.

Reply:
Yes, this is how it works and I think this should be the desired behavior. The GYM_OBS_KEY is used for filtering out the observations that are wanted in the Gym environment.